Makefile

```diff
@@ -15,7 +15,7 @@ RESET := $(shell tput -T linux sgr0)
 TITLE := $(BOLD)$(PURPLE)
 SUCCESS := $(BOLD)$(GREEN)
 # the quality gate lower threshold for unit test total % coverage (by function statements)
-COVERAGE_THRESHOLD := 60
```

Comment:
to be clear, this is unit test coverage threshold, not a holistic view of test coverage (integration tests should cover almost every part of the app)

Reply:
Apparently me adding some unit tests to the config package lowered the coverage because previously there hadn't been any unit tests there (which was ignored from the coverage report)